localenv/docker-compose.yaml

```diff
@@ -3,6 +3,15 @@
 version: "3.8"
 services:
 
+  anvil:
+    image: octant/anvil:latest
+    platform: linux/amd64
+    command: '--block-time "${BLOCK_TIME:-5}"'
+    ports:
+      - "8545:8545"
+    networks:
+      - octant
+
```

Comment:
What is the logic behind moving this into a base layer?

Reply:
Have you checked if "overloading" works when composing multiple compose files?

Reply:
I think I just went with solution I discovered (the composing) without checking if overloading works because of time constraints.

Reply:
After a check it turned out that overloading is not supported.